e2eshark/tools/aztestsetup.py

```diff
@@ -76,10 +100,47 @@ def download_onxx_model_from_azure_storage(cache_dir, testList):
 
         # TODO: better organisation of models in tank and cache
         print(
-            f"Downloading {blob_name} from {account_url}/{container_name} to {dest_file}"
+            f"Begin download for {blob_name} to {dest_file}"
         )
 
-        download_azure_blob(account_url, container_name, blob_name, dest_file)
+        try_private = False
+        try:
+            download_azure_blob(account_url, container_name, blob_name, dest_file)
+        except Exception as e:
+            try_private = True
+            print(f"Unable to download model from public for {model}.\nError - {type(e).__name__}")
+
+        if try_private:
+            print("Trying download from private storage")
+            try:
+                download_azure_blob(priv_account_url, priv_container_name, blob_name, dest_file)
+            except Exception as e:
+                print(f"Unable to download model for {model}.\nError - {type(e).__name__}")
+
+
+def pre_test_onnx_models_setup(testsList, cache_path, script_dir):
```

Comment:
Nit picking, but important to name identifiers to say what it does for productivity and keeping hygiene. Looks like this function downloads model too. Can have a more appropriate name?

Reply:
Sure, changed to pre_test_onnx_models_azure_download